packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -249,6 +253,16 @@ export class CallContext {
     });
   }
 
+  /* @conditional-compile-remove(rooms) */
+  public setRoleChanged(callId: string, role: ParticipantRole): void {
```

Comment:
Should this be named simply `setRole`?

`setRoleChanged` suggests that you're setting a callback of some sort.